### PR TITLE
Add a note for question 6 in the first homework.

### PR DESCRIPTION
**Pull request by SiamakMushakhian**
It is a suggestion for question 6 in the first home work.

### CODE REVIEW

File: course-zoomcamp/cohorts/2022/01-intro/homework.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ How many columns in the dataset have missing values?
 
 1. Find the median value of "Engine Cylinders" column in the dataset.
 2. Next, calculate the most frequent value of the same "Engine Cylinders".
-3. Use the `fillna` method to fill the missing values in "Engine Cylinders" with the most frequent value from the previous step.
+3. Use the `fillna` method to fill the missing values in "Engine Cylinders" with the most frequent value from the previous step. Please use 'inplace=Ture' to apply the filling method on the column.
```
Comment (alexeygrigorev) [inline course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: I wouldn't phrase it like that. In place is a possible option but not the only one

**Comment by alexeygrigorev** [comment]
@SVizor42 what do you think about this PR?

File: course-zoomcamp/cohorts/2022/01-intro/homework.md (hunk 1 above)
Comment (SiamakMushakhian) [reply course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: If the students don't use the "inplace=True", the filling will not apply to the data frame and in the following line, you asked them to calculate the median again. While if they don't do it in place, there is no difference between finding the median before and after filling (I know that in this case, there is no difference).

Comment (SVizor42) [reply course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: Hi, thanks for your comment!
I'm also sure that using `inplace=True` isn't mandatory. You can get the result without it.

Comment (SiamakMushakhian) [reply course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: Hi Timur,
Please do the following steps and then you will see the difference:
1- Load the data
df = pd.read_csv('./data/01-Car Price.csv')
2- Find the null rows for "Engine Cylinders"
df[df['Engine Cylinders'].isnull() == True]
3- Fill the NaN values with pd.fillna() without using inplace=True
df['Engine Cylinders'].fillna(df['Engine Cylinders'].mode())
4- Now, check the NaN values, for example, 1983. You will see that it is still NaN.
df.iloc[1983]

So, you should use inplace=True.

Comment (SVizor42) [reply course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: Siamak, 
Yes, in this case you can use the `inplace` option. I just wanted to say that you can also get a solution without it.

Comment (SiamakMushakhian) [reply course-zoomcamp/cohorts/2022/01-intro/homework.md:77]: I know, but the purpose of the course is learning and not getting a solution by chance. 
I don't insist to modify it, I found it helpful for all students. It is totally up to you to change it or not.

**Pull request closed without merging**